index.html

```diff
@@ -166,11 +169,56 @@ <h4>Skills</h4>
                 <li>Codepen</li>
                 <li>GitHub</li>
                 <li>GitLAb</li>
-                <li>Terminal</li>     
+                <li>Terminal</li>
             </ul>
         </section>
-        
+
+        <section id="greetings2">
+            <h1 class="GE">Get started</h1>
+            <p class="firstp">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Atque doloribus maiores,
+                rem excepturi.</p>
+
+                 <form class="form" action="#" target="" method="get" name="formDatosPersonales">
+                    <ul>
+                        <li><label for="text">Your name</label>
+                        <input type="text" name="nombre" id="nombre" required/>
```

Comment:
- [ ] please ensure your name input has `maxlength="30"` attribute. this will limit long names